x-pack/test/functional/page_objects/index_management_page.ts

```diff
@@ -16,6 +17,46 @@ export function IndexManagementPageProvider({ getService }: FtrProviderContext)
     async reloadIndicesButton() {
       return await testSubjects.find('reloadIndicesButton');
     },
+    async toggleRollupIndices() {
+      await testSubjects.click('checkboxToggles-rollupToggle');
+    },
+
+    async getIndexList() {
+      const indices = await testSubjects.findAll('indexTableRow');
```

Comment:
I think there's a much easier way to make one call on the table and get all the data in it returned in a neat object.  Some Cheerio function.  Maybe @dmlemeshko could elaborate?

I think you basically find the table element and then call `<tableElement>.parseDomContent();`